### ISSUE TITLE
Use http instead of htpps - not have "s"

### ISSUE THREAD

**Issue by duongvanle** [perliedman/leaflet-routing-machine#574]
I see that this lib send request to get direction data as https, like this:
https://router.project-osrm.org/route/v1/driving/105.8341598,21.0277644;105.9341598,21.0277644?overview=false&alternatives=true&steps=true&hints=;
The response is error. If i replace https to http, it work. Tried on the browser directly
Also, according to its document http://project-osrm.org/docs/v5.22.0/api/#requests, it use http, not https. Please help me this issue. Thanks

**Comment by Mehditeranee** [comment]
@duongvanle Indeed, the HTTPS certificate has expired and won't be renewed with the current configuration. If you're pointing your self-hosted frontend at OSRM, you'll need to use http://router.project-osrm.org/ (not https), or point it at your own OSRM server.

I am also looking for a solution to point to http link and not https.

@perliedman an idea?

**Comment by Mehditeranee** [comment]
@perliedman Indeed, it is a demo server, but the expiration of the SSL cert should not prevent its use for testing in a recipe environment.

**Comment by perliedman** [comment]
I think you can expect the demo server to stop working completely, as I understand it Mapbox has no interest in maintaining it any more, and no one else has indicated they want to take over maintenance.

Having said that, I think I could make that change if it helps.

**Comment by Mehditeranee** [comment]
@perliedman Yes, it would be very appreciable to make that change! 

I tried to use the serviceUrl parameter with the link "http://router.project-osrm.org/route/v1/" (without 's' for http) to see if that could do the trick, but it changes absolutely nothing. 
I get the same error.

<img width="794" alt="Capture d’écran 2020-04-07 à 17 30 32" src="https://user-images.githubusercontent.com/38721898/78688375-7e545c80-78f5-11ea-8c85-dc73079dfa1e.png">

**Comment by Mehditeranee** [comment]
Ok,  I found the trick.

**For those who are searching a solution for this :**

The link used for the demo server by leaflet is materializing as below: 
serviceUrl: 'https://router.project-osrm.org/route/v1'

So, we have to overwrite this line by adding this : 
serviceUrl: 'https://router.project-osrm.org/route/v1'

in :
